src/components/Verse/OverflowVerseActionsMenu.tsx

```diff
@@ -69,6 +70,12 @@ const OverflowVerseActionsMenu: React.FC<Props> = ({ verse }) => {
     });
   };
 
+  const onQuranReflectClicked = () => {
+    navigateToExternalUrl(
+      `https://quranreflect.com/${verse.chapterId}/${verse.verseNumber}?feed=true`,
+    );
```

Comment:
qq, why do we want to do this programmatically? Can we treat it like a regular link?